src/Plugin.php

```diff
@@ -175,6 +182,11 @@ public static function init() {
       add_filter('woocommerce_de_avoid_check_same_delivery_time_show_parent', '__return_true');
     }
 
+    add_action(self::CRON_EVENT_ENSURE_BACK_IN_STOCK, __NAMESPACE__ . '\WooCommerce::cron_ensure_back_in_stock');
+    if (!wp_next_scheduled(static::CRON_EVENT_ENSURE_BACK_IN_STOCK)) {
+      wp_schedule_event(strtotime('3am tomorrow'), 'daily', self::CRON_EVENT_ENSURE_BACK_IN_STOCK);
+    }
```

Comment:
1. If WooCommerce is not installed, we can skip the entire cron event.
2. Also update the cron event name constant accordingly to "remove" instead of "ensure".
3. Ditto for the callback function name.
4. The initial timestamp should be 3am today; otherwise you have to wait for one day until you see results and can confirm that it works.
```suggestion
    if (function_exists('wc_get_product')) {
      add_action(self::CRON_EVENT_REMOVE_BACK_IN_STOCK, __NAMESPACE__ . '\WooCommerce::cron_remove_back_in_stock');
      if (!wp_next_scheduled(static::CRON_EVENT_REMOVE_BACK_IN_STOCK)) {
        wp_schedule_event(strtotime('03:00:00'), 'daily', self::CRON_EVENT_REMOVE_BACK_IN_STOCK);
      }
    }
```